src/v3/src/components/AuthCoin/AuthCoinConfig.ts

```diff
@@ -39,91 +45,106 @@ const AuthenticatorConfiguration: {
-    description: 'DUO',
+    description: loc('factor.duo', 'login'),
     iconClassName: 'mfa-duo',
   },
   [AUTHENTICATOR_KEY.IDP]: {
     icon: IDPIcon,
+    name: 'idpAuthenticator',
     customizable: true,
-    description: 'IDP',
+    description: loc('factor.customFactor.description.generic', 'login'),
     iconClassName: 'mfa-custom-factor',
   },
   [AUTHENTICATOR_KEY.GOOGLE_OTP]: {
     icon: GoogleOTPIcon,
+    name: 'googleOtpAuthenticator',
     customizable: false,
-    description: 'Google Authenticator',
+    description: loc('factor.totpSoft.googleAuthenticator', 'login'),
     iconClassName: 'mfa-google-auth',
   },
   [AUTHENTICATOR_KEY.EMAIL]: {
     icon: EmailIcon,
+    name: 'emailAuthenticator',
     customizable: true,
-    description: 'Email',
+    description: loc('factor.email', 'login'),
     iconClassName: 'mfa-okta-email',
   },
   [AUTHENTICATOR_KEY.PASSWORD]: {
     icon: PasswordIcon,
+    name: 'passwordAuthenticator',
     customizable: true,
-    description: 'Password',
+    description: loc('factor.password', 'login'),
     iconClassName: 'mfa-okta-password',
   },
   [AUTHENTICATOR_KEY.OV]: {
     icon: OktaVerifyIcon,
+    name: 'oktaVerifyAuthenticator',
     customizable: false,
-    description: 'Okta Verify',
+    description: loc('factor.totpSoft.oktaVerify', 'login'),
     iconClassName: 'mfa-okta-verify',
   },
   [AUTHENTICATOR_KEY.ON_PREM]: {
     icon: OnPremMFAIcon,
+    name: 'onPremAuthenticator',
     customizable: true,
-    description: 'On Prem MFA',
+    description: loc('factor.totpHard.description', 'login'),
     iconClassName: 'mfa-onprem',
   },
   [AUTHENTICATOR_KEY.PHONE]: {
-    icon: PhoneCallIcon,
+    icon: PhoneIcon,
+    name: 'phoneAuthenticator',
     customizable: true,
-    description: 'Phone Call',
+    description: loc('factor.call', 'login'),
     iconClassName: 'mfa-okta-phone',
   },
   [AUTHENTICATOR_KEY.RSA]: {
     icon: RSAIcon,
+    name: 'rsaAuthenticator',
     customizable: false,
-    description: 'RSA',
+    description: loc('factor.totpHard.rsaSecurId', 'login'),
     iconClassName: 'mfa-rsa',
   },
   [AUTHENTICATOR_KEY.SECURITY_QUESTION]: {
     icon: SecurityQuestionIcon,
+    name: 'securityQuestionAuthenticator',
     customizable: true,
-    description: 'Security Question',
+    description: loc('factor.securityQuestion', 'login'),
     iconClassName: 'mfa-okta-security-question',
   },
   [AUTHENTICATOR_KEY.SYMANTEC_VIP]: {
-    icon: SymantecVipIcon,
+    icon: SymantecIcon,
+    name: 'symantecVipAuthenticator',
     customizable: false,
-    description: 'Symantec VIP',
+    description: loc('factor.totpHard.symantecVip', 'login'),
     iconClassName: 'mfa-symantec',
   },
   [AUTHENTICATOR_KEY.WEBAUTHN]: {
-    icon: WebAuthNIcon,
+    icon: SecurityKeyOrBiometricsIcon,
+    name: 'webauthnAuthenticator',
     customizable: true,
-    description: 'WebAuthN',
+    description: loc('factor.webauthn.biometric', 'login'),
```

Comment:
I see WebAuthn along with some factors have [separate "label" and "description" properties](https://github.com/okta/okta-signin-widget/blob/c5a41bb3de959dcd02cd97c04ef45f85a7cb6604/packages/@okta/i18n/src/properties/login.properties#L289-L290) (some have [variants with params](https://github.com/okta/okta-signin-widget/blob/c5a41bb3de959dcd02cd97c04ef45f85a7cb6604/packages/@okta/i18n/src/properties/login.properties#L286-L288)). Does this need to be handled for the `AuthCoin` or only for the `AuthenticatorButton`?

Reply:
Oh yes, those variations only need to be handled by the authenticator button (we actually have a custom i18n transformer that does exactly that). This is just for the `<Title>` element of the SVG, so when someone hovers over the SVG with a screenreader, it will read out whatever the Title contains.